doc/lightning-checkmessage.7.md

```diff
@@ -20,6 +20,10 @@ known node key (as per *listnodes*), and verification succeeds if it
 matches for any one of them.  Note: this is implemented far more
 efficiently than trying each one, so performance is not a concern.
 
+On failure, an error is returned and core lightning exit with the following error code:
+- -32602: Parameter missed or malformed;
+- 1301: *pubkey* not found in the graph.
+
```

Comment:
Edit the schema file, so that `verified` is show to always be `true`:

```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "type": "object",
  "required": [
    "verified",
    "pubkey"
  ],
  "additionalProperties": false,
  "properties": {
    "verified": {
      "type": "boolean",
      "enum": [
        true
      ],
      "description": "Whether the signature was valid"
    },
    "pubkey": {
      "type": "pubkey",
      "description": "the *pubkey* parameter, or the pubkey found by looking for known nodes"
    }
  }
}
```

This means you'll need to disable schema checks in test_signmessage, or update the test?

Reply:
(Maybe just update the test, by putting the guts of test_checkmessage_pubkey_not_found in there?

Reply:
Mh! Does the actual version of the test make sense? or do you want me to change it?